### DISCUSSION TITLE
Translate validation error count message

### DISCUSSION THREAD

**Discussion by sam1330** [laravel/framework#53108] [Ideas]
Hello! I've encountered a potential improvement to the message at the root of a validation error response. 
Context: I am developing a SaaS in Spanish and when there's more than one validation error, Laravel by default adds `(and :count more $pluralized)` and there's no easy way to translate this (could take the locale set in `app.php`). 

**Improvement**: It would be nice if we could translate or customize this message without creating weird methods or overriding existing ones like `failedValidation` in the `Illuminate\Foundation\Http\FormRequest` class.

NOTE: If there is any method that can do what I am explaining here, would be nice to know.

![imagen](https://github.com/user-attachments/assets/c1b99955-4c87-460c-a5c9-bc51a780627d)

**Comment by henzeb** [comment]
There have been several attempts, they don't want this fixed.

**Comment by flavio-schoute** [comment]
Have you tried this: https://dev.to/rodolfovmartins/validation-error-messages-in-laravel-customizing-and-localizing-feedback-1d4k 

Or: https://laravel.com/docs/11.x/validation#specifying-custom-messages-in-language-files

**Reply by sam1330** [reply]
It is a different problem. I haven't had any issues customizing the validation error messages, but the message in the root message that counts how many errors there are.
